SpatialGDK/Source/SpatialGDKEditor/Public/SpatialGDKEditorSettings.h

```diff
@@ -425,6 +425,17 @@ class SPATIALGDKEDITOR_API USpatialGDKEditorSettings : public UObject
 	UPROPERTY(EditAnywhere, config, Category = "Cloud Connection")
 	FString DevelopmentDeploymentToConnect;
 
+	/** Whether to start local server worker when connecting to cloud deployment. If selected, make sure that the cloud deployment you want to connect to is not automatically launching Server-workers. (That your workers are "manual_connection_only" type)*/
+	UPROPERTY(EditAnywhere, config, Category = "Cloud Connection", meta = (DisplayName = "Connect local server worker to the cloud deployment"))
+	bool bStartLocalServerWorker;
+
+	/** Port on which the receptionist proxy will be available.*/
+	UPROPERTY(EditAnywhere, config, Category = "Cloud Connection", meta = (EditCondition = "bStartLocalServerWorker", DisplayName = "Local Receptionist Port"))
+	int32 LocalReceptionistPort = 7777;
```

Comment:
Create a constant for the port. We do have a `DEFAULT_PORT` constant that's also 7777 but I'm not sure if using that here is a good idea in case it changes for some other reason, so I would prefer a separate constant for the external receptionist flow.

Reply:
Nit: Also initialize these in the constructor.